
2024-07-07 01:04:52.024575: W tensorflow/core/framework/local_rendezvous.cc:404] Local rendezvous is aborting with status: OUT_OF_RANGE: End of sequence
[34m[1mwandb[39m[22m:   101 of 101 files downloaded.
[34m[1mwandb[39m[22m:   101 of 101 files downloaded.
[34m[1mwandb[39m[22m:   101 of 101 files downloaded.
Epoch 1/10




[1m138/138[22m [32m━━━━━━━━━━━━━━━━━━━━[39m [1m63s[22m 405ms/step - accuracy: 0.9481 - loss: 0.1387 - val_accuracy: 0.9264 - val_loss: 0.1723
Epoch 2/10




[1m138/138[22m [32m━━━━━━━━━━━━━━━━━━━━[39m [1m48s[22m 346ms/step - accuracy: 0.9517 - loss: 0.1368 - val_accuracy: 0.9264 - val_loss: 0.1720
Epoch 3/10




[1m138/138[22m [32m━━━━━━━━━━━━━━━━━━━━[39m [1m50s[22m 363ms/step - accuracy: 0.9503 - loss: 0.1402 - val_accuracy: 0.9264 - val_loss: 0.1716
Epoch 4/10



[1m138/138[22m [32m━━━━━━━━━━━━━━━━━━━━[39m [1m47s[22m 346ms/step - accuracy: 0.9507 - loss: 0.1405 - val_accuracy: 0.9264 - val_loss: 0.1713
Epoch 5/10




[1m138/138[22m [32m━━━━━━━━━━━━━━━━━━━━[39m [1m52s[22m 378ms/step - accuracy: 0.9456 - loss: 0.1429 - val_accuracy: 0.9202 - val_loss: 0.1709